assets/js/blocks/featured-product/block.js

```diff
@@ -161,12 +174,18 @@ const FeaturedProduct = ( {
 					<ToggleControl
 						label={ __( 'Show description', 'woo-gutenberg-products-block' ) }
 						checked={ attributes.showDesc }
-						onChange={ () => setAttributes( { showDesc: ! attributes.showDesc } ) }
+						onChange={ () =>
+							setAttributes( { showDesc: ! attributes.showDesc } )
```

Comment:
I'd rather see the entire function on its own line rather than broken up like this OR break on the arguments not the function call.

Reply:
I moved everything to the next line and added a `prettier-ignore` rule in c26de1b9ffcba6813e36338c0dd17bcf06bc408e.